Source/WebCore/PAL/pal/PALSwift/CryptoKitShim.swift

```diff
@@ -66,7 +69,8 @@ public final class AesGcm: NSObject {
             if ivSize == 0 {
                 return .invalidArgument
             }
-            let nonce = try AES.GCM.Nonce(data: UnsafeBufferPointer(start: iv, count: Int(ivSize)))
+            let nonce = try AES.GCM.Nonce(
+                data: UnsafeBufferPointer(start: iv, count: Int(ivSize)))
```

Comment:
Our goal is to write Swift code that is at least consistent with some future vision for safety, even if Swift is not yet safe. If Nonce() is expressed as an unsafe pointer API, I think we want an explicit wrapper or extension or replacement for it, so that our code is expressed in terms of safety. Maybe we need a separate file called "CryptoKitUnsafeAPIWrappers" or something like that, to hold all these abstractions.

Reply:
This PR is only for Digest. I am going to cleanup the AESGCM, AESKW wrappers in a future PRs.